docker-compose.yml

```diff
@@ -111,10 +111,10 @@ services:
   # https://hub.docker.com/r/localstack/localstack/
   # localstack running a fake S3
   localstack-s3:
-    image: localstack/localstack:0.7.3
+    image: localstack/localstack:0.7.4
```

Comment:
I'm having difficulties with localstack losing data in the S3 container. Seems like data just disappears even if I'm not shutting down the container. I haven't been able to reproduce it at will, and I have some tools to work around it, so I've been ignoring it.

I thought updating to 0.7.4 might fix it, but it doesn't appear to have. I'll keep an eye on it and if it continues to be problematic, I may switch us to another system like minio or something that persists s3 contents to a volume.